### PR TITLE
Feat : UI  Changes in the Sign In page

### LINKED ISSUE #249: UI: Develop Sign In page
## Description
As a maintainer,
I need to see BIT Sign Up page gets redesigned,
so that I can make sure they meet AnitaB guidelines.

## Mocks
Approved design idea for Sign In page (Designer Gloria Osoro)
Screen Shot 2021-05-22 at 6 30 22 pm

### Update [Required]
- [ ]  Implement the design above in the section it is intended in the design above.

### Enhancement to Update [Optional]
NA

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 day

### PR DESCRIPTION
**Pull request by reachaadrika**
### Description
1. Updated the UI of Sign In Page
2. Created 2 new folders , login.jsx and login.css for the same

Fixes #249 

### Type of Change:

 Code

**Code/Quality Assurance Only**

 New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Screenshot (483)

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by stableapple** [comment]
@reachaadrika can you please resolve the conflict. You can refer to this [link](https://github.com/anitab-org/bridge-in-tech-web/pull/254#issuecomment-860574811). And please remove ``` package-lock.json``` file.

File: src/login/Login.jsx [hunk 1]
```diff
@@ -93,40 +100,31 @@ export default function Login() {
...
-                        </form-group>
-                        <div><br></br></div>
+                       
                         <div>
                             {errorMessage && <span className="error" name="errorMessage" aria-label="errorMessage" role="alert">{errorMessage}</span>}
                         </div>
+                        
                         <div className="row">
-                            <label>Not yet registered? Click on sign up.</label>
+                            <label>Dont have an Account? Sign Up.</label>
```
Comment (stableapple) [inline src/login/Login.jsx:107]: Link this to ```/register```

Comment (reachaadrika) [reply src/login/Login.jsx:107]: Yes Sure @stableapple 
Also can you please tell me how to remove package.json?

Comment (reachaadrika) [reply src/login/Login.jsx:107]: Can you please tell me how to do I commit latest code to this pull request only ?

Comment (stableapple) [reply src/login/Login.jsx:107]: To remove the ```package-lock.json``` you can do ```git reset --soft HEAD~1``` and then ```git rm package-lock.json```. To resolve the conflict, that is to merge the latest code from the master branch(develop) to your new branch that you are currently working in, you will have to change to the new branch by doing ```git checkout new_branch```. Then set upstream by ``` git remote add upstream https://github.com/anitab-org/bridge-in-tech-web.git``` followed by ```git pull upstream develop```

Comment (reachaadrika) [reply src/login/Login.jsx:107]: Thanks a lot 

git pull upstream develop
From https://github.com/anitab-org/bridge-in-tech-web
 * branch            develop    -> FETCH_HEAD
Already up to date.

This is what I am getting

**Comment by aakankshaagr** [comment]
@reachaadrika footer and show password is missing. Please add them too.

**Comment by reachaadrika** [comment]
Yes sure @aakankshaagr , please tell me once how to commit the latest code in this PR , I am facing some issues

**Comment by aakankshaagr** [comment]
sure @reachaadrika 
1. Switch to branch you created for solving this issue
2. add your latest changes (git add .)
3. commit them
4. push them 
You will be able to see that commit here

**Comment by reachaadrika** [comment]
Thank you so much @aakankshaagr

**Comment by reachaadrika** [comment]
@aakankshaagr  I tried to resolve conflicts but still they are n't going

**Comment by stableapple** [comment]
If you are facing issue using the command line, you can resolve the conflicts using the web editor by clicking on the ``` Resolve conflicts ```  button. And please create a separate branch,(other than the develop branch) while submitting your pull requests as this may cause problems in the future.

**Comment by reachaadrika** [comment]
@stableapple  Noted ,  Should I make a new PR?

**Comment by mtreacy002** [comment]
> @stableapple Noted , Should I make a new PR?

@reachaadrika , yes please 😉

**Comment by reachaadrika** [comment]
@mtreacy002  Sure , I will close this PR and make a new one

**Pull request closed without merging**